examples/simple/main.tf

```diff
@@ -53,7 +53,7 @@ module "materialize" {
   #     name           = "demo"
   #     namespace      = "materialize-environment"
   #     database_name  = "demo_db"
-  #     cpu_request    = "4"
+  #     cpu_request    = "2"
```

Comment:
In case people uncomment both, could with also have the `analytics` use 1 cpu_request?